acceptance/acceptance_test.go

```diff
@@ -80,16 +77,19 @@ func TestAccept(t *testing.T) {
 	user, err := workspaceClient.CurrentUser.Me(ctx)
 	require.NoError(t, err)
 	require.NotNil(t, user)
-	testdiff.PrepareReplacementsUser(t, &repls, *user)
-	testdiff.PrepareReplacements(t, &repls, workspaceClient)
+
+	ctx, repls := testdiff.WithReplacementsMap(ctx)
+	repls.Set(execPath, "$CLI")
+	testdiff.PrepareReplacementsUser(t, repls, *user)
+	testdiff.PrepareReplacements(t, repls, workspaceClient)
 
 	testDirs := getTests(t)
 	require.NotEmpty(t, testDirs)
 
 	for _, dir := range testDirs {
 		t.Run(dir, func(t *testing.T) {
 			t.Parallel()
-			runTest(t, dir, coverDir, repls)
+			runTest(t, ctx, dir, coverDir)
```

Comment:
We should not share ctx between different tests. 

Furthermore I don't think it's safe - ctx is passed by reference whereas ReplacementsContext is passed by value.

Reply:
I reverted this change and took a different approach through #2151.